<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LIN-G5PB1J3" failures="1" tests="5" name="Personal Loan" time="0.072" errors="0" timestamp="2024-04-03T16:56:50 IST">
  <testcase classname="test.Testing1" name="geturlname" time="0.009"/>
  <testcase classname="test.Testing1" name="logindata" time="0.004"/>
  <testcase classname="test.Testing1" name="logindata" time="0.001"/>
  <testcase classname="test.Testing1" name="logindata" time="0.001"/>
  <testcase classname="test.FailedTest" name="failedtestcase" time="0.005">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.FailedTest.failedtestcase(FailedTest.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- failedtestcase -->
</testsuite> <!-- Personal Loan -->
